convert2rhel/unit_tests/actions/system_checks/rhel_compatible_kernel_test.py

```diff
@@ -254,7 +254,7 @@ def test_bad_kernel_package_signature_success(
     monkeypatch,
     pretend_os,
 ):
-    run_subprocess_mocked = mock.Mock(spec=run_subprocess, return_value=(kernel_pkg, 0))
+    run_subprocess_mocked = RunSubprocessMocked(return_string=kernel_pkg)
```

Comment:
Seems that `get_installed_pkg_objects` is not being used

Reply:
Yeah... not sure why it was added in the first place (it was used like this originally: `mock.Mock(spec=get_installed_pkg_objects, [...])`) but it is definitely not needed now.

Removed.